### PR TITLE
Slack Code of Conduct and Usage Guidelines

### PR DESCRIPTION
**Pull request by hhariri**

### CODE REVIEW

File: pages/community/slackccugl.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+* #android. For Android generic topics
+* #beginner. For more beginner questions. Better ask here than in #general if you’re new to Kotlin
+* #communities. A place to discuss physical communities. Also search for cities/countries as there are often channels with these names (#australia), as well as languages (i.e. #spanish).
+* #coroutines. Questions about coroutines and async programming in Kotlin
+* #dagger. About the Dagger framework. 
+* #datascience. Using Kotlin for Data Science. 
+* #functional. Functional programming topics with Kotlin
+* #gradle. Working with Gradle and Kotlin Script Gradle
+* #intellij. Working with IntelliJ IDEA and Kotlin
```
Comment (yole) [inline pages/community/slackccugl.md]: People will interpret this as saying that it's OK to ask general IntelliJ questions there. I'd really prefer that they don't

Comment (hhariri) [inline pages/community/slackccugl.md]: We can rephrase it as "Regarding the Kotlin plugin for IntelliJ IDEA" which is the purpose of the channel. Or alternatively remove it, but the channel is there.

Comment (yole) [inline pages/community/slackccugl.md]: Yes, let's rephrase.

**Comment by hhariri** [comment]
Done

**Pull request merged**